integration-tests/smoke/vrfv2plus_test.go

```diff
@@ -1000,23 +1001,25 @@ func TestVRFV2PlusWithBHS(t *testing.T) {
 		mockETHLinkFeed,
 		numberOfTxKeysToCreate,
 		1,
-		1,
+		2,
 		l,
 	)
 	require.NoError(t, err, "error setting up VRF v2_5 env")
 
-	subID := subIDs[0]
-
-	subscription, err := vrfContracts.CoordinatorV2Plus.GetSubscription(testcontext.Get(t), subID)
-	require.NoError(t, err, "error getting subscription information")
-
-	vrfv2plus.LogSubDetails(l, subscription, subID, vrfContracts.CoordinatorV2Plus)
-	var isNativeBilling = false
+	var isNativeBilling = true
 	t.Run("BHS Job with complete E2E - wait 256 blocks to see if Rand Request is fulfilled", func(t *testing.T) {
 		t.Skip("Skipped since should be run on-demand on live testnet due to long execution time")
+
+		subID := subIDs[0]
+
+		subscription, err := vrfContracts.CoordinatorV2Plus.GetSubscription(testcontext.Get(t), subID)
+		require.NoError(t, err, "error getting subscription information")
+
+		vrfv2plus.LogSubDetails(l, subscription, subID, vrfContracts.CoordinatorV2Plus)
```

Comment:
put this snippet in a dedicated function? seems that it's copy & pasted at least 2x

Reply:
not sure about which snippet part you are referring to?